sw/host/tests/manuf/manuf_scrap/src/main.rs

```diff
@@ -40,13 +40,15 @@ fn manuf_scrap(opts: &Opts, transport: &TransportWrapper) -> Result<()> {
         "Invalid initial LC state.",
     );
 
+    // FIXME: unclear
```

Comment:
I think this one should be fine since CPU execution is disabled, though it is interesting LC TAP works, but perhaps this is intended

CC: @msfschaffner